runtime/RuntimeCommon.cpp

```diff
@@ -219,3 +219,13 @@ void symcc_make_symbolic(void *start, size_t byte_length) {
   _sym_make_symbolic(start, byte_length, inputOffset);
   inputOffset += byte_length;
 }
+
+SymExpr _sym_build_bits_to_bool(SymExpr expr) {
```

Comment:
I have two remarks on this function:
1. Since we only call the function on `i1` expressions, should we rename it to `_sym_build_bit_to_bool` (for symmetry with the inverse)?
2. Do we need to implement it in terms of the new `_sym_build_ite`? Wrapping in ITE seems unnecessary because the type of the equality check is already a Boolean. We could also leave the implementation to the backends, just like `_sym_build_bool_to_bit`, and use the QSYM expressions' `boolToBit` method; I don't know that it has any advantages though, so an implementation in `RuntimeCommon.cpp` is also fine with me.